CardIOView.ios.js

```diff
@@ -1,4 +1,5 @@
-import React, { Component, PropTypes } from 'react';
+import PropTypes from  'prop-types';
```

Comment:
nit: Can you swap the imports? I prefer the `React` import to be on top of the file.